stable/nextcloud/README.md

```diff
@@ -150,14 +150,19 @@ The following table lists the configurable parameters of the nextcloud chart and
 | `hpa.maxPods`                                                | Max. pods for the Nextcloud HorizontalPodAutoscaler     | `10`                                        |
 | `deploymentAnnotations`                                      | Annotations to be added at 'deployment' level           | not set                                     |
 | `podAnnotations`                                             | Annotations to be added at 'pod' level                  | not set                                     |
-| `metrics.enabled`                                            | Start a single Prometheus metrics exporter              | `false`                                     |
-| `metrics.https`                                              | Wether https is used for scraping Nextcloud metrics     | `false`                                     |
+| `metrics.enabled`                                            | Start Prometheus metrics exporter                       | `false`                                     |
+| `metrics.https`                                              | Defines if https is used to connect to nextcloud        | `false` (uses http)                         |
```

Comment:
@mrueg Does this edit of the documentation improves the clarity?

Reply:
Yup, thanks!